charts/flyte-core/values-gcp.yaml

```diff
@@ -313,10 +313,17 @@ configmap:
 cluster_resource_manager:
   # -- Enables the Cluster resource manager component
   enabled: true
-  # -- Starts the cluster resource manager in standalone mode with requisite auth credentials to call flyteadmin service endpoints
-  standalone_deploy: false
+  # -- Service account name to run with
+  service_account_name: flyteadmin
+  # -- Annotations for ClusterResource pods
+  podAnnotations: {}
   config:
     cluster_resources:
+      # -- How frequently to run the sync process
+      refreshInterval: 5m
+      templatePath: "/etc/flyte/clusterresource/templates"
+      # -- Starts the cluster resource manager in standalone mode with requisite auth credentials to call flyteadmin service endpoints
+      standaloneDeployment: false
```

Comment:
Do we need these here if they are already in the values.yaml file?

Reply:
oh right, reverted!